src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -42,10 +42,10 @@ public class Arm extends SubsystemBase {
     public static final double EXTENDER_CALIBRATION_MOTOR_SPEED = 0.1d;
 
     /* Conversion Factors */
-    public static final double ENCODER_UNITS_PER_REVOLUTION = 2048.0d / 1.0d;
-    public static final double PIVOT_GEARBOX_MOTOR_TO_GEARBOX_ARM_RATIO = 36.00d / 1.0d;
+    public static final double ENCODER_UNITS_PER_REVOLUTION = 2048.0d;
+    public static final double PIVOT_GEARBOX_MOTOR_TO_GEARBOX_ARM_RATIO = 36.00d;
     public static final double PIVOT_PULLEY_MOTOR_TO_PULLEY_ARM_RATIO = 72.0d / 36.0d;
-    public static final double EXTENDER_GEARBOX_MOTOR_TO_GEARBOX_ARM_RATIO = 4.0d / 1.0d;
+    public static final double EXTENDER_GEARBOX_MOTOR_TO_GEARBOX_ARM_RATIO = 4.0d;
```

Comment:
Let's leave the `/ 1.0d` in there to make it very clear that these ratios are 4 : 1 etc.

Reply:
Done